js/analyses/getSubheadings.js

```diff
@@ -0,0 +1,25 @@
+/* @module analyses/getSubheadings */
+
+var stripSomeTags = require( "../stringProcessing/stripNonTextTags.js" );
+var subheadingMatch = require( "../stringProcessing/subheadingsMatch.js" );
+
+/**
+ * Checks if there are any subheadings like h2 in the text
+ * and if they have the keyword in them.
+ *
+ * @param {string} text The text to check for subheadings.
+ * @returns {object} the result object.
```

Comment:
We should document what specific object we are returning instead of just `{object}`